packages/block-library/src/social-link/socials-without-bg.scss

```diff
@@ -155,6 +155,10 @@
 	color: #3499cd;
 }
 
+.wp-social-link-x {
```

Comment:
I'm not getting the color on the front-end. Do you mind confirming this works? 

<img width="210" alt="CleanShot 2023-09-11 at 16 22 51" src="https://github.com/WordPress/gutenberg/assets/1813435/483bbc25-d294-450a-9038-7440c6ca0a95">

Reply:
It seems to test well for me (this is Twenty Twenty Three):

**Editor**

<img width="1388" alt="image" src="https://github.com/WordPress/gutenberg/assets/426388/633ffda0-20d0-4c37-adac-920edd298c05">

**Frontend**

<img width="703" alt="image" src="https://github.com/WordPress/gutenberg/assets/426388/31ea3ae6-6012-42e8-9a1a-06cabccbb5b6">

Reply:
It's working fine for me as well.